### PR TITLE
PR: Improve versions of debug icons, add new cell icon and optimize svg icons

### PR DESCRIPTION
**Pull request by conradolandia**
## Description of Changes

* [x] Improved icons for Debug operations (`debug`, `debug_cell`, `debug_line`) both in Dark and Light versions. 
* [x] Added new icon for Debug (generic).
* [x] Added new icon for `new_cell`.

I've added two replacements for icons `debug_cell` and `debug_selection`, plus a new generic `debug` icon. Another icon was added for `new_cell`. 

Visualization simulation

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:

Andres Montoya

### CODE REVIEW

File: spyder/images/dark/new_cell.svg
Comment (ccordoba12) [inline spyder/images/dark/new_cell.svg:1]: I'm not sure if we should use magenta as the color for this icon. Instead, I think we could simply show it in white for the dark theme and in black for the light one.

What do you think?

**Review by ccordoba12 — COMMENTED** [review]
Thanks @conradolandia for your work on this!

And congratulations for your first PR in Spyder!!

File: spyder/images/dark/new_cell.svg
Comment (CAM-Gerlach) [reply spyder/images/dark/new_cell.svg:1]: Yeah, at least IMO the magenta color is a little more distracting to me and I'm not sure I understand the motivation given that cells aren't shown as magenta by default.

**Review by CAM-Gerlach — COMMENTED** [review]
The changes generally look good to me (I've long advocated for swapping the pause and play positions), though I do agree with @ccordoba12 's comments with regard to the color of the cell icon—see my comment there.

File: spyder/images/dark/new_cell.svg
Comment (ccordoba12) [reply spyder/images/dark/new_cell.svg:1]: Good point too: the current magenta draws too much attention to that icon unnecessarily.

Comment (conradolandia) [reply spyder/images/dark/new_cell.svg:1]: Agreed. Honestly, that was the original color of the "alpha version" icon and I didn't think about changing it, but you guys are right.

**Comment by conradolandia** [comment]
I've changed the `new_cell` icon color for both Dark and Light versions. Also, optimized all the SVG files on both the Dark and Light folders, I managed to get files around 30%-50% of the original size, without compromising looks (as far as I can see). However, please check them out, and let me know if you notice something weird.

**Comment by ccordoba12** [comment]
@conradolandia, I reviewed your PR locally and it needs some minor additional changes to work as expected:

1. To load the new debug icon you need to remove this line in our code:

    https://github.com/spyder-ide/spyder/blob/dc90e8fa1bb0559846f0e01639e6e43a59c385bc/spyder/utils/icon_manager.py#L141

    and this old icon in our source code: `spyder/images/editor/debug.png`

2. To load the new cell icons you need to rename them to `cell.svg` (both for the dark and light themes) and remove this other line in our code:

    https://github.com/spyder-ide/spyder/blob/dc90e8fa1bb0559846f0e01639e6e43a59c385bc/spyder/utils/icon_manager.py#L322

----

In addition, I noticed that the new `debug_cell` icon doesn't have the right color for the cell frame in the dark theme, as can be seen in the screenshot below:

imagen

but that's a quite simple fix.

----

You also said:

> Also, optimized all the SVG files on both the Dark and Light folders, I managed to get files around 30%-50% of the original size, without compromising looks (as far as I can see). However, please check them out, and let me know if you notice something weird.

Great work @conradolandia! I checked them at high resolution and I see no difference with the previous icons.

**Comment by conradolandia** [comment]
Hi guys, so... I got a different idea for the icon. I know this was kind of "solved" but I kinda wasn't feeling it with the `%%`. 

This new version is more coherent for the rest of the icon system, regarding what we visually refer as a "new cell". I created it with no color bits for now, but we can discuss that. Maybe some color on the `+` sign? 

What do you think?

New cell icon, version 2

**Comment by CAM-Gerlach** [comment]
Yeah, that's a great idea and a lot more visually consistent to me with both the appearance and symbology of the other icons. Agreed :+1:

**Comment by ccordoba12** [comment]
I totally agree with @CAM-Gerlach. That icon looks quite nice in our toolbar along the others, so great work @conradolandia!

Please upload it instead of the current cell icon so we can merge this PR.

**Comment by conradolandia** [comment]
Done!

**Review by ccordoba12 — APPROVED** [review]
Thanks @conradolandia for your work on this!

Note: There's a test slot that started to hang but it's unrelated to the changes here, so I'm going to merge it.

**Pull request merged**